container/container.go

```diff
@@ -436,17 +436,25 @@ func (w *ContainerWorker) finalizeDeployment(agreementId string, deployment *con
 		// the cgoup permission can be omitted. It defaults to "rwm" when omitted.
 		for _, givenDevice := range service.Devices {
 			cgp := "rwm"
+			pic := ""
 			sp := strings.Split(givenDevice, ":")
 			if len(sp) == 3 {
 				// the cgroup permission
 				cgp = sp[2]
-			} else if len(sp) < 2 || len(sp) > 3 {
+				pic = sp[1]
+			} else if len(sp) == 2 {
+				cgp = cgp
+				pic = sp[1]
+			}else if len(sp) < 1{
+				cgp = cgp
```

Comment:
this line is redundant because you have set it to "rwm" at line 438

Reply:
Ohh. Added by mistake!